src/utils/deploy/hash-fns.ts

```diff
@@ -195,4 +219,25 @@ const hashFns = async (
   return { functionSchedules, functions, functionsWithNativeModules, fnShaMap, fnConfig }
 }
 
+const trafficRulesConfig = (trafficRules: $TSFixMe) => {
```

Comment:
I would be nice to import the type that we'll expose to users and use it here, but we can do that as a follow-up.

Reply:
in https://github.com/netlify/cli/pull/6503/commits/8f4a538e1b497b94d6c5cf575d1d29141836abd2, i've imported the TrafficRules type from the ZISI internals. Is that what you had in mind? Also, do we consider importing those internals criminal?

Reply:
Yes, exactly! I don't think it's criminal, but I think it would be good to properly export them so we're not relying on internal file paths.

Reply:
agreed. i'll make the zisi PR and replace the import in a followup.

Reply:
here's said PR: https://github.com/netlify/zip-it-and-ship-it/pull/1724